CMakeLists.txt

```diff
@@ -12,7 +12,11 @@ option( BUILD_SHARED_LIBS             "Build exiv2lib as a shared library"
 option( EXIV2_ENABLE_XMP              "Build with XMP metadata support"                       ON  )
 option( EXIV2_ENABLE_EXTERNAL_XMP     "Use external version of XMP"                           OFF )
 option( EXIV2_ENABLE_PNG              "Build with png support (requires libz)"                ON  )
+if ( MSVC )
```

Comment:
An idea to improve this is to use  **CMakeDependentOption**. In that way we will avoid to have code duplication. This would turn into something like this:

```
cmake_dependent_option(EXIV2_ENABLE_NLS   "Build native language support (requires gettext)" ON "MSVC" OFF)
```

Reply:
That's beautiful.  Much much nicer.  I think the CMake GUI will understand that.  My change probably causes him stress!